yt_dlp/extractor/tvp.py

```diff
@@ -482,21 +482,32 @@ def _real_extract(self, url):
 class TVPVODBaseIE(InfoExtractor):
     _API_BASE_URL = 'https://vod.tvp.pl/api/products'
 
-    def _call_api(self, resource, video_id, **kwargs):
+    def _call_api(self, resource, video_id, query={}, **kwargs):
         return self._download_json(
             f'{self._API_BASE_URL}/{resource}', video_id,
-            query={'lang': 'pl', 'platform': 'BROWSER'}, **kwargs)
+            query={'lang': 'pl', 'platform': 'BROWSER', **query}, **kwargs)
 
-    def _parse_video(self, video):
-        return {
-            '_type': 'url',
-            'url': 'tvp:' + video['externalUid'],
-            'ie_key': TVPEmbedIE.ie_key(),
+    def _parse_video(self, video, with_url=True):
+        info_dict = {
+            'id': str(video['id']),
             'title': video.get('title'),
-            'description': traverse_obj(video, ('lead', 'description')),
+            'description': clean_html(video.get('lead') or video.get('description')),
```

Comment:
Old code was looking at `video.load.description`. Is this change intentional? Was the old code wrong? Don't tests change accordingly?

```suggestion
            'description': traverse_obj(video, 'lead', 'description', expected_type=clean_html),
```

Reply:
it was supposed to be lead or description, tests apparently had both so far and it worked